tests/gen/fp-add/add_200.expect

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What are these new tests for?

Reply:
These are to check that changing the gen config parameters actually does change the generated verilog; the tests check that the latency of the module changes with different frequencies etc.

Reply:
Got it!